pkgs/applications/networking/p2p/transmission/4.nix

```diff
@@ -141,8 +141,14 @@ stdenv.mkDerivation rec {
     EOF
   '';
 
+  passthru.tests = {
+    apparmor = nixosTests.transmission_4; # starts the service with apparmor enabled
+    smoke-test = nixosTests.bittorrent;
```

Comment:
Is not this testing Transmission 3?

Reply:
this is transmission_4, I assume we want to test transmission_4 here